### ISSUE TITLE
What to set in AWS-EXECUTION_ENV and where to set it?

### ISSUE THREAD

**Issue by shubhisood** [typless/tesseract-aws-lambda#1]
Great Article!! I have followed all the steps mentioned by you but for me, lambda is giving this error while executing oc_as_service.py file:
`{
  "errorMessage": "Unable to import module 'handler': No module named 'Image'",
  "errorType": "Runtime.ImportModuleError"
}`
For now I have set AWS_EXECUTION_ENV to Layer's ARN, Does anything else I am missing in this?

Thanks for sharing such a great article, Please help me in resolving this issue.

**Comment by jangia** [comment]
Did you install PIL?

**Comment by shubhisood** [comment]
yes it is installed within layer named ocr which is linked to lambda.
Following is the folder structure for layer:
![image](https://user-images.githubusercontent.com/47347277/91208657-23969280-e728-11ea-8dfb-e9135e8656bf.png)

**Comment by jangia** [comment]
I'd do following things:
* Check that you have python3.7 set as runtime in serverless config
* Check that your function is really using layer (list /opt/python/lib/python3.7/site-packages/ on lambda function - use console with test)
